src/data/actions/assets.test.jsx

```diff
@@ -159,15 +206,72 @@ describe('Assets Action Creators', () => {
     expect(store.dispatch(actionCreators.uploadExceedMaxCount(1))).toEqual(expectedAction);
   });
   it('returns expected state from uploadAssetFailure', () => {
-    const expectedAction = { file: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
+    const expectedAction = { asset: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
     expect(store.dispatch(actionCreators.uploadAssetFailure('file', 'response'))).toEqual(expectedAction);
   });
   it('returns expected state from uploadAssetSuccess', () => {
-    const expectedAction = { asset: 'asset', response: 'response', type: assetActions.UPLOAD_ASSET_SUCCESS };
-    expect(store.dispatch(actionCreators.uploadAssetSuccess('asset', 'response'))).toEqual(expectedAction);
+    const expectedAction = { response: 'response', type: assetActions.UPLOAD_ASSET_SUCCESS };
+    expect(store.dispatch(actionCreators.uploadAssetSuccess('response'))).toEqual(expectedAction);
   });
   it('returns expected state from uploadingAssets', () => {
     const expectedAction = { count: 99, type: assetActions.UPLOADING_ASSETS };
     expect(store.dispatch(actionCreators.uploadingAssets(99, 'response'))).toEqual(expectedAction);
   });
+  it('returns expected state from uploadAssets success', () => {
+    const assetsResponse = {
+      status: 200,
+      body: {
+        assets: ['a.txt'],
+      },
+    };
+
+    fetchMock.mock(`begin:${assetsEndpoint}`, getUploadResponse(true), { method: 'post' });
```

Comment:
The uploadAssets action creator makes two types of fetch calls: one post to upload the asset and one get to get the new assets from the server. These two requests should have different responses, so I've set up the fetch-mock to discriminate them based on whether or not they're post or get requests.